plugins/pencilblue/templates/admin/content/articles/article_form.html

```diff
@@ -7,12 +7,26 @@
 		<fieldset>
 			<div class="tab-content">
 				<div class="tab-pane active" id="content">
+					<div class="form-group" ng-class="{'has-error': !isFieldValid(articleForm.headline)}">
```

Comment:
Since url will be generated from the headline, it is only natural to first define the headline then use it in url generation.